Vercel adapter default changes (#8239)
withastro/astro: .changeset/silly-dolphins-try.md

---
'@astrojs/vercel': major
---

> Vercel adapter now defaults to a function per route

Reviewer: ```suggestion
Vercel adapter now defaults to `functionPerRoute`.
```

> With this change, in `@astrojs/vercel/serverless`, now each route is split into its own function. By doing this the size of each function is reduced and startup time is faster.

Reviewer 2: ```suggestion
With this change, `@astrojs/vercel/serverless` now splits each route into its own function. By doing this, the size of each function is reduced and startup time is faster.
```
Take a comma, leave a comma!

> You can disable this option, which will cause the code to be bundled into a single function, by setting `functionPerRoute` to false.

Reviewer: ```suggestion
You can disable this option, which will cause the code to be bundled into a single function, by setting `functionPerRoute` to `false`.
```
